package.json

```diff
@@ -51,18 +51,18 @@
     "recompose": "^0.27.0"
   },
   "devDependencies": {
-    "@storybook/react": "^4.0.0-alpha.4",
+    "@storybook/react": "^3.4.6",
```

Comment:
why are we downgrading storybook ? If I remember correctly, there is a bug with webpack 4 and storybook.